docs/pages/hooks/index.mdx

```diff
@@ -0,0 +1,172 @@
+---
+title: 'Introduction'
+---
+
+# Liteflow SDK
+
+This is the documentation for the Liteflow SDK. The SDK contains a set of React Hooks that facilitates the creation of your own NFT Marketplace.
```

Comment:
Liteflow SDK is all the packages, not just the hook, right?
```suggestion
# Hooks

This is the documentation for the Liteflow's hooks. This package contains a set of React Hooks that facilitates the creation of your own NFT Marketplace.
```

Reply:
Yes just moved this text, need to update

Reply:
done